openapi/openapi.yaml

```diff
@@ -1080,6 +1080,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackageMetadataResponseError"
 
+  /python/get-package-from-imports:
```

Comment:
get is already in the `GET` method. Might be a good idea to rename this to something like `/python/imports`.